Makefile

```diff
@@ -23,30 +25,28 @@ build/build-docker-image.stamp: Dockerfile
 
 build-docker-image: build/build-docker-image.stamp
 
-build/download-packages.stamp: build/build-docker-image.stamp src/download_packages.sh
-	mkdir -p build/packages
+build/download-packages.stamp: build/build-docker-image.stamp src/compilation/download_packages.sh
+	mkdir -p $(BUILD_PACKAGES_DIR)
 	docker run --user $(shell id -u):$(shell id -g) \
 		--rm --volume .:/app/gdb gdb-static env TERM=xterm-256color \
-		/app/gdb/src/download_packages.sh /app/gdb/build/packages
+		/app/gdb/src/compilation/download_packages.sh /app/gdb/$(BUILD_PACKAGES_DIR)/
 	touch build/download-packages.stamp
 
-download-packages: build/download-packages.stamp
+build/symlink-git-packages.stamp: $(SUBMODULE_PACKAGES)
```

Comment:
we are going to need to refactor this one day. i guess its fine for now...

Reply:
The actual commit/branch is set in the repository, not in the submodule file.

Reply:
Yeah, we do. I am not a very good Makefile builder. This is mostly copied from the other action make rules.